documentation/architecture.drawio

```diff
@@ -1,20 +1,20 @@
-<mxfile host="app.diagrams.net" modified="2022-03-03T06:58:21.012Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36" etag="jawmesXoiLuioSDJKZNp" version="16.6.4" type="github">
+<mxfile host="app.diagrams.net" modified="2022-03-31T09:36:37.073Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36" etag="lK_5tippkE5IevXrBJ4l" version="17.1.3" type="github">
```

Comment:
Blaue Box. Ist ein "Reverse Proxy" nicht schon der Begriff für Inbound Kommunikation? 
Also: "Dataland Inbound Reverse Proxy" should be "Dataland Reverse Proxy" OR "Dataland Inbound Proxy"

Reply:
Oki doki, danke für den Link.